diff --git a/node_modules/react-native-phone-number-input/lib/index.js b/node_modules/react-native-phone-number-input/lib/index.js
index b89c388..0259582 100644
--- a/node_modules/react-native-phone-number-input/lib/index.js
+++ b/node_modules/react-native-phone-number-input/lib/index.js
@@ -1,13 +1,13 @@
+import { PhoneNumberUtil } from "google-libphonenumber";
 import React, { PureComponent } from "react";
-import { View, Text, TouchableOpacity, Image, TextInput } from "react-native";
+import { Image, Text, TextInput, TouchableOpacity, View } from "react-native";
 import CountryPicker, {
-  getCallingCode,
+  CountryModalProvider,
   DARK_THEME,
   DEFAULT_THEME,
-  CountryModalProvider,
   Flag,
+  getCallingCode,
 } from "react-native-country-picker-modal";
-import { PhoneNumberUtil } from "google-libphonenumber";
 import styles from "./styles";
 
 const dropDown =
@@ -191,7 +191,7 @@ export default class PhoneInput extends PureComponent {
               disableNativeModal={disabled}
               visible={modalVisible}
               theme={withDarkTheme ? DARK_THEME : DEFAULT_THEME}
-              renderFlagButton={this.renderFlagButton}
+              // renderFlagButton={this.renderFlagButton}
               onClose={() => this.setState({ modalVisible: false })}
               {...countryPickerProps}
             />
